dist/GaugeChart/index.js

```diff
@@ -113,46 +146,31 @@ var GaugeChart = function GaugeChart(props) {
     window.addEventListener("resize", handleResize);
     return function () {
       window.removeEventListener("resize", handleResize);
-    };
+    }; // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [props]);
   var id = props.id,
       style = props.style,
-      className = props.className;
+      className = props.className,
+      textComponent = props.textComponent,
+      textComponentContainerClassName = props.textComponentContainerClassName;
   return /*#__PURE__*/_react.default.createElement("div", {
     id: id,
     className: className,
     style: style,
     ref: function ref(svg) {
       return selectedRef = svg;
     }
-  });
+  }, /*#__PURE__*/_react.default.createElement("div", {
+    className: textComponentContainerClassName,
+    style: {
+      position: "relative",
+      top: "50%"
+    }
+  }, textComponent));
```

Comment:
as mentioned in the PR description, these changes are from previously merged commits that did not include a corresponding update to the build output.